packages/contracts/test/plugin/plugin-setup-processor.ts

```diff
@@ -250,43 +299,63 @@ describe('Plugin Setup Processor', function () {
           psp.prepareInstallation(
             targetDao.address,
             pluginSetupBad,
-            pluginSetupMockRepoAddress,
+            repoUUPS.address,
             data2
           )
         ).to.be.revertedWith(customError('SetupAlreadyPrepared'));
       });
 
-      it('retruns correctly the plugin, helpers, and permissions', async () => {
-        const {plugin, helpers, permissions} =
-          await psp.callStatic.prepareInstallation(
+      it('returns the plugin, helpers, and permissions', async () => {
+        let plugin;
+        let helpersV1;
+        let permissionsV1;
+        ({
+          plugin: plugin,
+          helpers: helpersV1,
+          permissions: permissionsV1,
+        } = await prepareInstallation(
+          psp,
+          targetDao.address,
+          setupV1.address,
+          repoUUPS.address,
+          EMPTY_DATA
+        ));
+
+        expect(plugin).to.not.equal(AddressZero);
+        expect(helpersV1).to.deep.equal(mockHelpers(1));
+        expect(permissionsV1).to.deep.equal(
+          mockPermissionsOperations(0, 1, Operation.Grant).map(perm =>
+            Object.values(perm)
+          )
+        );
+      });
+
+      it('prepares a UUPS upgradeable plugin installation', async () => {
+        await expect(
+          prepareInstallation(
+            psp,
             targetDao.address,
-            pluginSetupV1Mock.address,
-            pluginSetupMockRepoAddress,
+            setupV1.address,
+            repoUUPS.address,
             EMPTY_DATA
-          );
+          )
+        ).to.not.be.reverted;
+      });
 
-        expect(plugin).not.to.be.equal(AddressZero);
-        expect(helpers.length).to.be.equal(1);
-        expect(permissions).to.deep.equal([
-          [
-            Op.Grant,
+      it('prepares a cloneable plugin installation', async () => {
+        await expect(
+          prepareInstallation(
+            psp,
             targetDao.address,
-            plugin,
-            AddressZero,
-            ethers.utils.id('EXECUTE_PERMISSION'),
-          ],
-          [
-            Op.Grant,
-            plugin,
-            helpers[0],
-            AddressZero,
-            ethers.utils.id('SETTINGS_PERMISSION'),
-          ],
-        ]);
+            setupC1.address,
+            repoClones.address,
+            EMPTY_DATA
+          )
+        ).to.not.be.reverted;
```

Comment:
check that it was initialized correctly with the correct variable..

Reply:
We have a test for the initialization already